docs/subnets/customize-a-subnet.md

```diff
@@ -436,6 +437,79 @@ In addition to the AllowList interface, the FeeConfigManager adds the following
 - `getFeeConfigLastChangedAt` - retrieves the timestamp of the last block where the fee config was updated
 - `setFeeConfig` - sets the dynamic fee config on chain (see [here](#fee-config) for details on the fee config parameters)
 
+### Network Upgrades: Enable/Disable Precompiles
+
+Disclaimer: performing a network upgrade requires coordinating the upgrade network-wide. A network upgrade changes the rule set used to process and verify blocks, such that any node that upgrades incorrectly or fails to upgrade by the time that upgrade goes into effect may become out of sync with the rest of the network.
+
+In addition to specifying the configuration for each of the above precompiles in the genesis chain config, they can be individually enabled or disabled at a given timestamp as a network upgrade. Disabling a precompile disables calling the precompile and destructs its storage so it can be enabled at a later timestamp with a different configuration if desired.
+
+These upgrades can be specified in a file named `upgrade.json` placed in the same directory as `config.json` using the following format:
+
+```json
+{
+  "precompileUpgrades": [
+    {
+      "<precompileName>": {
+        "blockTimestamp": 100, // timestamp precompile should activate at
```

Comment:
is this unix timestamp? If so, we should put in better example value such as 1658944144

Reply:
They are unix timestamps.

Reply:
Let me update the doc with better examples.

Reply:
Done